package.json

```diff
@@ -38,10 +38,11 @@
 	"license": "MIT",
 	"dependencies": {
 		"about-window": "^1.15.2",
+		"micropython.js": "github:murilopolese/micropython.js#v1.2.2",
```

Comment:
so now the `micropython.js` is a module brought in from your repo?
interesting :)

Reply:
For now, I would propose so. In the future it should be a npm package under the Arduino organization, I believe.